packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -364,6 +393,9 @@ export class Combobox
   /** Specifies the size of the component. */
   @property({ reflect: true }) scale: Scale = "m";
 
+  /** When `true`, provides a toggle for selecting all items. Does not apply to `selection-mode single`. */
```

Comment:
doc consistency recommendation 📚:

```suggestion
  /** When `true` and `selectionMode` is `"multiple"` or `"ancestors"`, provides a checkbox for selecting all `calcite-combobox-item`s. */
```